spec/factories/repayments.rb

```diff
@@ -5,7 +5,7 @@
     amount_paid { amount_due }
     interest_since_last_payment { rand(1..amount_due) }
     date_due { Faker::Date.between(Date.civil(2004, 01, 01), Date.today) }
-    loan_id { create(:loan).id }
+    project_id { create(:loan).id }
```

Comment:
Yeah, repayments were an incomplete feature from the old Cofunder front end. Let's leave them alone for now.